preprocessing/preprocess_data.sh

```diff
@@ -112,60 +112,86 @@ cd ${SUBJECT}/anat
 # We do a substitution '/' --> '_' in case there is a subfolder 'ses-0X/'
 file="${SUBJECT//[\/]/_}"
 
-# Add suffix corresponding to contrast
-file=${file}_acq-sag_T2w
+# Add suffix corresponding to contrast (Sagittal T2w)
+file_sag=${file}_acq-sag_T2w
 
-# Make sure the image metadata is a valid JSON object
-if [[ ! -s ${file}.json ]]; then
-  echo "{}" >> ${file}.json
+# Add suffix corresponding to contrast (Axial T2w)
+file_ax=${file}_acq-ax_T2w
+
+# Make sure the Sagittal image metadata is a valid JSON object
+if [[ ! -s ${file_sag}.json ]]; then
+  echo "{}" >> ${file_sag}.json
+fi
+
+# Make sure the Axial image metadata is a valid JSON object
+if [[ ! -s ${file_ax}.json ]]; then
+  echo "{}" >> ${file_ax}.json
 fi
 
-# Spinal cord segmentation using the T2w contrast
-segment_if_does_not_exist ${file} t2 ${CENTERLINE_METHOD}
-file_seg="${FILESEG}"
+# Resample both images to isotropic 1mm x 1mm x 1mm resolution
+flirt -in ${file_sag}.nii.gz -ref ${file_sag}.nii.gz -applyisoxfm 1.0 -nosearch -out ${file_sag}_res.nii.gz
```

Comment:
why adding this dependency? SCT can also do it (sct_register_multimodal -identity 1)

Reply:
how about `sct_resample` here instead? This is because I am trying to register the original axial to the iso resampled sagittal and for doing that the `-d` and `-dseg` params for `sct_register_multimodal` need the resampled sagittal image. I am not sure how using just the `-identity 1` makes sure that the sagittal image is also resampled. It doesn't right?

Reply:
yup! `sct_resample` would work